controllers/etcdcluster_controller_test.go

```diff
@@ -273,12 +273,18 @@ func (s *controllerSuite) testClusterController(t *testing.T) {
 			// Assert on peers
 			peers := &etcdv1alpha1.EtcdPeerList{}
 			err = try.Eventually(func() error {
-				return s.k8sClient.List(s.ctx, peers, &client.ListOptions{
+				err := s.k8sClient.List(s.ctx, peers, &client.ListOptions{
 					Namespace: namespace,
 				})
+				if err != nil {
+					return err
+				}
+				if len(peers.Items) != 3 {
+					return fmt.Errorf("wrong number of peers: %#v", len(peers.Items))
```

Comment:
We know it's an `int`, so is `%d` clearer then using `%#v` and letting the default formatter figure it out?

 ```suggestion
					return fmt.Errorf("wrong number of peers: %d", len(peers.Items))
```

Reply:
Also should we say what the expected number of peers actually is in the error message?

Reply:
Done. Error message now looks like this:
 
```
...
            log.go:23: log.V(2).WithName("EtcdCluster").Info: Unable to contact etcd cluster -- [cluster exotic-chow/cluster1 error unable to connect to etcd: fake etcd, nothing is here]
            --- FAIL: TestAPIs/ClusterControllers/OnCreation/CreatesPeers (5.00s)
                etcdcluster_controller_test.go:291: 
                    	Error Trace:	etcdcluster_controller_test.go:291
                    	Error:      	Received unexpected error:
                    	            	wrong number of peers. expected: 3, actual: 3
                    	Test:       	TestAPIs/ClusterControllers/OnCreation/CreatesPeers
...
```